src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -116,7 +116,21 @@ public function denormalize($data, $class, $format = null, array $context = [])
         }
 
         try {
-            return \DateTime::class === $class ? new \DateTime($data, $timezone) : new \DateTimeImmutable($data, $timezone);
+            $datetime = new \DateTime($data, $timezone);
+            /**
+             * PHP docs say when created a new DateTime object:.
+             *
+             * The $timezone parameter and the current timezone are ignored when the $time parameter either is a UNIX timestamp (e.g. @946684800) or specifies a timezone (e.g. 2010-01-28T15:00:00+02:00).
+             *
+             * This means the behaviour in this normalizer is different when the timezone is applied in the constructor or later with setTimezone.
```

Comment:
Pleas use “behavior”

Reply:
Hi @OskarStark,  Is there a language guideline for Symfony?  I refer English (Oxford) over America (Webster).  Is American just your preference?